$NetBSD: patch-bj,v 1.1.2.2 2009/01/28 19:18:36 tron Exp $

Add patch from http://bugs.mysql.com/file.php?id=9232,
referenced on http://bugs.mysql.com/bug.php?id=27884, to fix the
vulnerability recorded in
http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-4456.

--- mysql-test/t/mysql.test.orig	2008-08-04 12:32:17.000000000 +0000
+++ mysql-test/t/mysql.test
@@ -290,4 +290,12 @@ EOF
 --exec $MYSQL < $MYSQLTEST_VARDIR/tmp/bug29323.sql 2>&1
 remove_file $MYSQLTEST_VARDIR/tmp/bug29323.sql;
 
+
+#
+# Bug #27884: mysql --html does not quote HTML special characters in output
+# 
+--exec $MYSQL --html test -e "select '< & >' as \`<\`"
+
+--echo
+--echo End of 5.1 tests
 --echo End of 5.0 tests
